code/go/0chain.net/smartcontract/stakepool/lock.go

```diff
@@ -113,7 +113,7 @@ func (sp *StakePool) LockPool(
 		Reward:       currency.Coin(0),
 		Total:        i,
 	}
-	balances.EmitEvent(event.TypeStats, event.TagLockStakePool, newPoolId, lock)
+	balances.EmitEvent(event.TypeStats, event.TagLockStakePool, txn.ClientID, lock)
```

Comment:
if client have different stake pools, those events will be merged even for different pools, is it the behavior we want?